package.json

```diff
@@ -97,20 +107,26 @@
         "chai": "^4.2.0",
         "chalk": "^4.0.0",
         "changelog-parser": "^2.8.0",
+        "chokidar": "^3.5.1",
```

Comment:
So, you're gonna laugh at my flip-flip on this "issue". but with the way you have the extension coded, it will fail without chokidar being a dependency, since the import happens at the top of the file. We can negate that by changing the code to use chokidar like this inside your `if` block:
```
if (isDebugMode) {
    const chokidar = require('chokidar');
    ...other logic
}
```

But it's probably just easier to switch this back to a dependency. :D